redbot/core/dev_commands.py

```diff
@@ -92,6 +93,28 @@ def sanitize_output(ctx: commands.Context, input_: str) -> str:
         token = ctx.bot.http.token
         return re.sub(re.escape(token), "[EXPUNGED]", input_, re.I)
 
+    def get_environement(self, ctx: commands.Context) -> dict:
```

Comment:
This is spelled incorrectly. Remember to also update any usage of this method.
```suggestion
    def get_environment(self, ctx: commands.Context) -> dict:
```